### DISCUSSION TITLE
Lockdown App

### DISCUSSION THREAD

**Discussion by guikaneto** [Ylianst/MeshCentral#5890] [General]
I can connect, but when open an app like lockdown browser i can see only the primary desktop, i can’t the lockdown browser window.

someone have this problem?

thk.

**Comment by silversword411** [comment]
That's literally it's entire purpose. To be an isolated app, that isn't viewable and...locked down.

**Comment by guikaneto** [comment]
i know, but see some people can be view this window. How?
